src/actions/types.ts

```diff
@@ -95,7 +82,7 @@ export interface IStateQueryString {
  */
 
 export interface ActionPushAlert {
-    type: typeof PUSH_ALERT;
+    type: string;
```

Comment:
Was it necessary to widening the type for action?

Reply:
These interfaces have now been dropped in favour of narrower types for the payloads only.

Reply:
Sorry, I do not understand, maybe we can setup online meeting?
Or maybe there is an example that covers this change?
Is it related to transition in major version from 7 to 8?